Formula/xray.rb

```diff
@@ -41,6 +48,22 @@ def install
     resource("geosite").stage do
       pkgshare.install "dlc.dat" => "geosite.dat"
     end
+
+    resource("example_config").stage do
+      pkgetc.install "config.json"
+    end
```

Comment:
```suggestion
    pkgetc.install resource("example_config")
```
I think this should do the same thing; can you check this locally? We may want to simplify the lines that install other resources too if it does.

Other than this, this LGTM

Reply:
passed local tests